odoo/tools/profiler.py

```diff
@@ -147,13 +147,30 @@ class SQLCollector(Collector):
     name = 'sql'
 
     def start(self):
+        current_thread = threading.current_thread()  # don't store current_thread on self
+        self.explain_sql = getattr(current_thread, 'profiler_params', {}).get('explain_sql')
```

Comment:
Why do you need the current_thread to access params? they should be on `self.profiler.params` no ? Also for sql collector the current_thread should always be the sames as the init_thread if I remember well.